acquisitions/src/main/resources/thunderjet/mod-orders/features/check-holding-instance-creation-with-createInventory-options.feature

```diff
@@ -0,0 +1,88 @@
+# MODORDERS-902
+Feature: check-holding-instance-creation-with-createInventory-options
+
+  Background:
+    * url baseUrl
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * configure headers = headersUser
+
+    * callonce variables
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+
+
+  Scenario:  Prepare finances
```

Comment:
Probably it is better to do all data initialization in Background in general, so Scenarios are not dependent on each other

Reply:
Fixed it, thanks